### PR TITLE
Fix validation loss logging

### PR DESCRIPTION
**Pull request by eantono**
## Motivation

The validation loss is currently not logged in the setting of both train and val workflow modes.

The problem is described in this issue:
https://github.com/open-mmlab/mmsegmentation/issues/1396
This PR replicates the changes made in this PR to the mmsegmentation library: https://github.com/open-mmlab/mmsegmentation/pull/1494

## Modification

`val_step` is modified to add `_val` suffix to all validation losses, to distinguish those losses from those in the training step. This enables both training and validation loss logging.

### CODE REVIEW

**Review by ZwwWayne — APPROVED** [review]

**Comment by jason102811** [comment]
Hi @eantono  ！First of all, we want to express our gratitude for your significant PR in the MMDet. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/UjgXkPWNqA

If you have WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤
